site/content/en/docs/Installation/upgrading.md

```diff
@@ -128,7 +128,94 @@ upgrades.
    between the Agones controller being recreated and GameServers being deleted doesn't occur, and GameServers can end up stuck in erroneous states.
 1. Start and complete your control plane upgrade(s).
 1. Start and complete your node upgrades.
-1. Scale your Fleets back up and/or recreate your GameServers. 
+1. Scale your Fleets back up and/or recreate your GameServers.
 1. Run any other tests to ensure the Agones installation is still working as expected.
 1. Close your maintenance window.
 7. Congratulations - you have now upgraded to a new version of Kubernetes! 👍
+
+{{% feature publishVersion="1.41.0" %}}
+## SDK Compatibility Guarantees
+
+The SDK compatibility contract aims to ensure smooth upgrades and reduce the need for frequent
+binary redeployments for game server users.
+What’s the point of having a compatibility guarantee?
+Goal: Reduce user toil by minimizing the need for frequent binary redeployments due to SDK changes.
+Versioning: Clear documentation of API maturity levels and history (Stable, Beta, Alpha) with release information.
+
+Our SDK Server compatibility contract as of Agones v1.41.0: A
+game server binary using Beta and Stable SDKs will remain compatible with a _newer_ Agones Release,
+within possible deprecation windows:
+- If your game server uses a non-deprecated Stable API, your binary will be compatible for 10
+releases (~1y) starting from the SDK version packaged.
```

Comment:
```suggestion
releases (~1y) or more starting from the SDK version packaged.
```

WDYT? It read like it'll stop working after 1y (even though there is a line below it).